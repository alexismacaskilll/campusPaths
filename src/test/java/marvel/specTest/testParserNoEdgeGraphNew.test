# Test the parser on a file that contains 3 heros from different comics

# Load a graph from tsv file that contains 3 heros from different comics
LoadGraph noEdgeGraph noEdgesNew.tsv

# Print the heros in the graph
ListNodes noEdgeGraph

# Print the outgoing edges from each hero in the graph
ListChildren noEdgeGraph hero_1
ListChildren noEdgeGraph hero_2
ListChildren noEdgeGraph hero_3