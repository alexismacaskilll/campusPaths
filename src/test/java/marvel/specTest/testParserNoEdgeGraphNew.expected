# Test the parser on a file that contains 3 heros from different comics

# Load a graph from tsv file that contains 3 heros from different comics
loaded graph noEdgeGraph

# Print the heros in the graph
noEdgeGraph contains: hero 1 hero 2 hero 3

# Print the outgoing edges from each hero in the graph
the children of hero 1 in noEdgeGraph are:
the children of hero 2 in noEdgeGraph are:
the children of hero 3 in noEdgeGraph are: